CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+# v1.6.8
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* Add option to retrieve pprof traces from running cilium-agents (Backport PR #10684, Upstream PR #10666, @aanm)
+* Update k8s libraries to 1.16.8 (#10662, @aanm)
+
+**Bugfixes:**
+* cilium: encryption, additional mtu fix for non-default 1500B MTU (Backport PR #10638, Upstream PR #10551, @jrfastab)
```

Comment:
The PRs backported by https://github.com/cilium/cilium/pull/10638 are *not* in this release as we are skipping past that PR.